tests/test_trade.py

```diff
@@ -15,7 +15,7 @@
 class BaseTradeTest(unittest.TestCase):
     """Generic Trade Test class"""
 
-    def run_base_trade_test(self, policy):
+    def run_base_trade_test(self, policy, additional_overrides=None):
```

Comment:
where is this actually used?

Reply:
in notebook_test, to allow quick testing in the notebook only, without having to edit the config file, which also runs the risk of having to revert it when you're done, or forgetting what you've changed